source/manual/document-collection-emails-a-special-case.html.md

```diff
@@ -0,0 +1,72 @@
+section: Emails
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+---
+
+[Document Collections][document_collections_schema] are published by Whitehall, and rendered by [government-frontend][government-frontend]. The majority of document_collection pages allow users to sign up to email notifications via a [single page notification button][single_page_notification_button_component_guide]. Clicking this button subscribes users to email notifications when a major change is published for the page, or content reverse linked to the page.
+
+For a small number of document_collections published in late 2023, the email signup feature works differently. These differences were agreed as part of negotiations with a specific government department to unblock the retirement of specialist topics.
+
+[document_collections_schema]: /document-types/document_collection.html
+[single_page_notification_button_component_guide]: https://components.publishing.service.gov.uk/component-guide/single_page_notification_button
+[government-frontend]: https://github.com/alphagov/government-frontend
+
+## Document collections with a taxonomy_topic_email_override link
+
+If the document_collection content item contains a populated `taxonomy_topic_email_override` field, government-frontend renders an [email sign up link][email_signup_component_guide] (rather than the single page notification button) for the taxonomy topic that matches the content_id stored in this field. In other words, a user is subscribed to the existing SubscriberList for the taxonomy topic, rather than being subscribed to the document_collection SubscriberList.
+
+**Note:** Part of the roll out of this feature was ensuring that all document_collections with a `taxonomy_topic_email_override` were also tagged to the same taxonomy topic. This ensures subscribers are notified in the event of the document_collection page itself being updated.
+
+[email_signup_component_guide]: https://components.publishing.service.gov.uk/component-guide/signup_link
+
+### How to set a taxonomy_topic_email_override
+
+**Note: The following steps should only be taken if the request comes via a content designer that has followed the guidance set out in the [2nd line - GOV.UK content - government support documentation][content_documentation].**
+
+[content_documentation]: https://gov-uk.atlassian.net/wiki/spaces/CC/pages/138674387/2nd+line+-+government+support+queue
+
+This field can only be set for document_collections that have never been published before.
+
+#### How to check if a document_collection has previously been published
+
+1. Find the document_collection in the whitehall admin UI
+2. Find the ID. E.g. `12345` for a document_collection at URL `whitehall-admin.publishing.service.gov.uk/government/admin/collections/12345`
+3. `DocumentCollection.find(12345).document.live?` returns `true` for previously published document_collections.
+
+#### How to set a taxonomy topic email override field for new pages
```

Comment:
I believe "finding the document collection id" is a necessary step for both "checking if the document collection has been published" and to "set the taxonomy topic override".

However the why the steps are laid out makes it look you always need to check if a document collection has been published first, but I believe the rake task checks this for you, and raises an exception.

Perhaps have the "How to check if a document_collection has previously been published" steps ask people to run the rake task in "dry run" mode to check for errors rather than spinning up a rails console?

I'd also duplicate the step for finding the document collection id before the instructions for running the rake task.

Reply:
Ah think I've not addressed the last sentence here so I won't resolve this comment!

Reply:
I don't think you need to anymore. It's clear now that these are steps one and two, so finding the ID needs to be done first.